t/pod.t

```diff
@@ -0,0 +1,15 @@
+#!perl -T
+use 5.006;
+use strict;
+use warnings;
+use Test::More;
+
+unless ($ENV{RELEASE_TESTING}) {
+    plan(skip_all => "Author tests not required for installation");
+}
+
+# Ensure a recent version of Test::Pod
+my $min_tp = 1.22;
+eval "use Test::Pod $min_tp";
+
+all_pod_files_ok();
```

Comment:
please make this test unconditional on travis. That means install all the prerequisites and remove all `skip_all`. Otherwise, nobody will run them.